src/components/schedule-page/validation-drawer/validation-drawer.component.tsx

```diff
@@ -1,5 +1,4 @@
-import { Drawer, List, ListItem, ListItemIcon, ListItemText } from "@material-ui/core";
-import ErrorOutlineIcon from "@material-ui/icons/ErrorOutline";
+import { Drawer } from "@material-ui/core";
```

Comment:
Potrzeba będzie zmienić import, ponieważ my już mamy nasz `Drawer`.
https://github.com/Project-Summer-AI-Lab-Glider/nurse-scheduling-problem-frontend/blob/97aac84890af4d6344b8bd1ba2ef51ab303cccea/src/components/common-components/drawer/drawer.component.tsx#L11

To musi sprowadzić się do podmiany importu. Jeżeli nie - to pisz i będziemy nad tym kminić

Reply:
Co powinno być tytułem?
![title](https://user-images.githubusercontent.com/48785655/101992330-ae3cd680-3cb2-11eb-9276-d451f5b7c6bf.png)

Reply:
Bo jakiś muszę podać:
![drawer2](https://user-images.githubusercontent.com/48785655/101992342-cc0a3b80-3cb2-11eb-910d-415654a7217f.png)

Reply:
Proponuję "Znalezione błędy"

Reply:
Mi się wydaje, że lepiej będzie `Sprawdź plan`, bo nie zawsze w grafiku będą błędy